src/api.js

```diff
@@ -39,30 +43,10 @@ async function inject(filename, resourceName, resourceData, options) {
 
   switch (executableFormat) {
     case postject.ExecutableFormat.kMachO:
-      {
-        let sectionName = resourceName;
-
-        // Mach-O section names are conventionally of the style __foo
-        if (!sectionName.startsWith("__")) {
-          sectionName = `__${sectionName}`;
-        }
-
-        ({ result, data } = postject.injectIntoMachO(
-          executable,
-          machoSegmentName,
-          sectionName,
-          resourceData,
-          overwrite
-        ));
-
-        if (result === postject.InjectResult.kAlreadyExists) {
-          throw new Error(
-            `Segment and section with that name already exists: ${machoSegmentName}/${sectionName}\n` +
-              "Use --overwrite to overwrite the existing content"
-          );
-        }
-      }
-      break;
+      execSync(
+        `/usr/local/opt/llvm/bin/llvm-objcopy --add-section ${machoSegmentName},__${resourceName}=${resource} ${filename}`
```

Comment:
can we just use `llvm-objcopy` from `$PATH`?

Reply:
`brew install llvm` does not add it to the path

Reply:
hmm. this will probably not work on linux, my system the llvm install is located under `/usr/lib/llvm/14/`

Reply:
correct, the path (even the binary name) is different on linux. I haven't implemented that part yet, will do that soon.